lib/core/request.js

```diff
@@ -79,7 +86,7 @@ class Request {
       this.body = body.byteLength ? body : null
     } else if (typeof body === 'string') {
       this.body = body.length ? Buffer.from(body) : null
-    } else if (util.isIterable(body) || util.isBlobLike(body)) {
+    } else if (util.isFormDataLike(this.body) || util.isIterable(body) || util.isBlobLike(body)) {
```

Comment:
I don't think `this.body` is defined yet so it will never be a FormData, and `util.isIterable` returns true for FormData which is why passing a FormData previously didn't throw

Reply:
Thanks!